_rules/device-motion-bf0de1.md

```diff
@@ -0,0 +1,327 @@
+---
+id: bf0de1
+name: No functions are exclusively operated through device motion
+rule_type: composite
+description: |
+  This rule checks that functionality that can be operated by device motion or user motion can also be operated by user interface components and responding to the motion can be disabled to prevent accidental actuation
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_rules:
+  - 5d9784
+  - de951c
+
+authors:
+  - João Vicente
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document](https://dom.spec.whatwg.org/#concept-document).
+
+## Expectation
+
+For each test target, the outcome of all of the following rules is passed:
+
+- [Motion can be replaced by user interface components](https://act-rules.github.io/rules/5d9784)
+- [Motion can be disabled](https://act-rules.github.io/rules/de951c)
+
+## Assumptions
+
+- This test assumes that motion is not [essential](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-essential) for the function
+
+## Accessibility Support
+
+- Device orientation events may include sensitive data. Implementations must fire events only on [secure browsing contexts](https://www.w3.org/TR/secure-contexts/)
+
+## Background
+
+- [Understanding Success Criterion 2.5.4: Motion Actuation](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html)
+- [G213: Provide conventional controls and an application setting for motion activated input](https://www.w3.org/WAI/WCAG21/Techniques/general/G213.html)
+- [DeviceOrientation Event Specification](https://www.w3.org/TR/orientation-event/)
+- [Detecting device orientation](https://developer.mozilla.org/en-US/docs/Web/API/Detecting_device_orientation)
+- [DeviceMotionEvent](https://developer.mozilla.org/en-US/docs/Web/API/DeviceMotionEvent)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+[HTML document](https://dom.spec.whatwg.org/#concept-document) does not have [functionality](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-functionality) that can be operated by device motion or user motion
+
+```html
+<div>document content</div>
+```
+
+#### Passed Example 2
+
+[HTML document](https://dom.spec.whatwg.org/#concept-document) has [functionality](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-functionality) that can be operated by device motion or user motion and can also be operated by [user interface components](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html#dfn-user-interface-component) and can be disabled
+
+```html
+<html>

```

Comment:
you are going to get a lot of test failures because the composed HTML is not complete. Please refer to other rules in the repository where we ignore some checks done by `_rules/__tests__/testcases.js`

Please ensure you run npm test locally and ignore what needs to be ignored or fix accordingly.

Eg: see - https://github.com/act-rules/act-rules.github.io/blob/develop/_rules/html-has-title-2779a5.md#L21